drivers/age-swift/README.md

```diff
@@ -0,0 +1,3 @@
+# age-swift
```

Comment:
Perhaps you can add a quick start guide (Swift usage), with links to AGE quick start (for setting up db).

Reply:
Alright I will do that in my next pull request

Reply:
Once the readme is set up, I will comment on the usage and perhaps have some suggestions for the correct level of abstraction for a driver - it should be easy to use in the most common scenarios, possible in the edge cases. 

Great work so far!